tss-esapi/src/constants/tss.rs

```diff
@@ -644,6 +644,10 @@ pub const TSS2_RESMGR_TPM_RC_LAYER: TSS2_RC = 0x000C0000; /* base is a TPM2_RC_*
 
 pub const TSS2_RC_SUCCESS: TSS2_RC = 0x00000000;
 
+pub const TPMI_DH_SAVED_TRANSIENT: TPMI_DH_SAVED = 0x80000000; /* an ordinary transient object */
```

Comment:
It's too bad that these comments are not doc comments above the constants but at least the change is consistent :sweat_smile:

Reply:
I mean we could add the same comment to the Enum item if that would make things better.